apps/web/lib/components/Kanban.tsx

```diff
@@ -55,23 +59,15 @@ function headerStyleChanger(snapshot: DraggableStateSnapshot, bgColor: any) {
  * @param param0
  * @returns
  */
-function InnerItemList({
-	items,
-	title,
-	dropSnapshot
-}: {
-	title: string;
-	items: ITeamTask[];
-	dropSnapshot: DroppableStateSnapshot;
-}) {
+function InnerItemList({ items, title }: { title: string; items: ITeamTask[]; dropSnapshot: DroppableStateSnapshot }) {
 	return (
 		<>
 			<section
-				style={{
-					minHeight: items.length < 0 && dropSnapshot.isDraggingOver ? '120px' : '20px',
-					marginTop: items.length > 0 ? '20px' : '0px'
-				}}
-				className="flex flex-col gap-2.5  overflow-x-hidden"
+				// style={{
```

Comment:
@Anishali2 why this code is commented? It was doing something to set height / margin top dynamically etc, why we don't need it anymore? Do you understand what was purpose of that code before you removed it?